specs/paymentrequest.html

```diff
@@ -682,9 +689,21 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
           <p>If the sequence only contains one item, then this is the shipping option that
           will be used and <a><code>shippingOption</code></a> will be set to the <code>id</code>
           of this option without running the <a>shipping option changed algorithm</a>.</p>
+          <p>If an item in the sequence has the <code>selected</code> field set to <code>true</code>,
+          then this is the shipping option that will be used by default and <a><code>shippingOption</code></a>
+          will be set to the <code>id</code> of this option without running the <a>shipping option changed
+          algorithm</a>. Authors SHOULD NOT set <code>selected</code> to <code>true</code> on more than
+          one item. If more than one item in the sequence has <code>selected</code> set to <code>true</code>,
+          then <a>user agents</a> MUST select the last one in the sequence.</p>
```

Comment:
There are two ways to read this line, so it's slightly unclear if the UAs should select the last one in the sequence of shipping options, or the last one in the sequence /with the selected flag set to true/. I think you mean the latter.